Admin/Admin.php

```diff
@@ -292,7 +292,7 @@
      * then the $parentReflectionProperty must be the ReflectionProperty of
      * the order (OrderElement::$order).
      *
-     * @var \ReflectionProperty
+     * @var mixed
```

Comment:
Uuuuuh… where does this reflection property stuff come from? Isn't this variable supposed to be a string?